docs/antrea-network-policy.md

```diff
@@ -502,8 +572,21 @@ or drops traffic which matches all `from`, `ports` sections.
 Under `ports`, the optional field `endPort` can only be set when a numerical `port`
 is set to represent a range of ports from `port` to `endPort` inclusive.
 `protocols` defines additional protocols that are not supported by `ports`. Currently, only
-ICMP protocol is under `protocols`. `icmpType` and `icmpCode` could be used to specify the ICMP
-traffic that this rule matches.
+ICMP protocol and IGMP protocol are under `protocols`. For `ICMP` protocol, `icmpType`
+and`icmpCode` could be used to specify the ICMP traffic that this rule matches. And
+for `IGMP` protocol, `igmpType` and `groupAddress` can be used to specify the IGMP
+traffic that this rule matches. If `igmpType` is not set, all reports will be matched.
+If `groupAddress` is empty, then all multicast group addresses will be matched here.
+Currently, only IGMP reports are supported in egress rules. Protocol `IGMP` can not
```

Comment:
Too many "currently". Maybe just remove this one.

Reply:
done